frontend/views/containers/payments/PaymentRowSent.vue

```diff
@@ -106,6 +114,14 @@ export default ({
         console.error(e)
         alert(e.message)
       }
+    },
+    async updatePaymentRelativeTo (paymentDate) {
+      this.relativeTo = await this.historicalPeriodStampGivenDate(paymentDate)
+    }
+  },
+  watch: {
+    payment (to, from) {
+      this.updatePaymentRelativeTo(to.date)
```

Comment:
Same comment/concern

Reply:
Isn't there any way a PaymentRowSent can be updated, for example in case of partial payment update?

Reply:
@snowteamer I don't remember. However, even if that were the case it only updates the total sent, not the period it was sent in I don't think... Feel free to test it out manually with the two approaches and see if there's a difference. I bet there won't be.

Reply:
Also is it safe to assign an async handler to `mounted`?

Reply:
Yes I’m pretty sure you can write `async mounted () {`, there might even be some existing examples in the project (not at my computer at the moment)